playwright.config.ts

```diff
@@ -33,6 +39,7 @@ export default defineConfig({
       use: { ...devices['Desktop Firefox'] },
     },
 
+    // Webkit disabled for now due to a bug
```

Comment:
does webkit work now?

Reply:
yes, passed in all 3

Reply:
fabulous, I assume this is just part of the cleanup then